### PR TITLE
ring chart edits:

### LINKED ISSUE #72: color of ring chart is confusing
screen shot 2017-07-19 at 9 08 24 pm
(for apn=3717/005)
the ring shows red, and there is no mention as to what "32 out of 100" means.

### LINKED ISSUE #96: ring chart code in own file
- [x] put the code for the ring chart into its own file
- [x] put it into its own scope like histogram-chart and quartiles-chart

### PR DESCRIPTION
**Pull request by thfield**
lower ranking means more filled in, code now in separate chart module, color scale reversed
fixes #96
along with #73, will address #72 

screen shot 2017-08-07 at 7 15 00 pm

### CODE REVIEW

**Comment by askiefer** [comment]
Actually @thfield maybe we should keep the ring chart and put the ranking (#9 out of 23) inside it for clarity (the #9 can be larger font on a separate line maybe). What do you think?

**Comment by thfield** [comment]
I'm conflicted in that Ii like having a ring chart but that isn't a good reason to hold it so precious....

I'll make that change and see what it looks like, maybe it'll be worth keeping around

(related to #98)

**Comment by thfield** [comment]
quick and dirty version: 
screen shot 2017-08-10 at 1 48 20 pm

**Comment by askiefer** [comment]
Thanks @thfield! I don't mind it but will defer to you and @eayoungs. What do you both think?

**Comment by thfield** [comment]
let's implement #98 ~instead of~ on top of this PR for now;

feedback from DOE: 
Change out circle graphic to something similar to:
```
#5
Out of 15 Buildings
```

File: dashboard/dashboard.js [hunk 1]
```diff
@@ -499,13 +473,16 @@ function populateInfoBoxes (singleBuildingData,categoryData,floorAreaRange) {
   d3.select('#building-energy-star-score').text(singleBuildingData.latest_energy_star_score)
   d3.selectAll('.building-energy-star-score-year').text(singleBuildingData.latest_energy_star_score_year)
   d3.select('#building-eui').text(singleBuildingData.latest_site_eui_kbtu_ft2)
-  d3.selectAll('.building-ghg-emissions').text(singleBuildingData.latest_total_ghg_emissions_metric_tons_co2e)
-  d3.selectAll('.building-ghg-emissions-year').text(singleBuildingData.latest_total_ghg_emissions_metric_tons_co2e_year)
```
Comment (thfield) [inline dashboard/dashboard.js]: add this line back in

**Comment by thfield** [comment]
fixes #101

**Review by askiefer — APPROVED** [review]
This adds the text to the ring chart so going to go ahead and approve - it looks like we're planning on merging 98 too, based on your comment?

**Comment by thfield** [comment]
yeah, I think we should implement your solution in 98, which just displays the ranking without the ring chart.  However, I want to first merge in this PR so the work of separating out the ring chart gets moved along and we can reuse it if we want a different ring chart somewhere

**Pull request merged**